randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -872,7 +872,7 @@ Extra - map_name: 7975C
 Extra - asset_id: 28
 > Door to Climb; Heals? False; Default Node
   * Layers: default
-  * Normal Door to Climb/Door to Pit Room
+  * Locked until Crateria Wakeup to Climb/Door to Pit Room
```

Comment:
Can you remove the "TODO: make left door of after crateria wakeup type" comment from right door -> left door, and also make the connection trivial?

Reply:
Done